src/org/traccar/protocol/SuntechProtocolEncoder.java

```diff
@@ -25,6 +25,13 @@
     protected Object encodeCommand(Command command) {
 
         switch (command.getType()) {
+            case Command.TYPE_REBOOT_DEVICE:
+                return formatCommand(command, "SA200CMD;{%s};02;Reboot\r", Command.KEY_UNIQUE_ID);
+            case Command.TYPE_POSITION_SINGLE:
+                return formatCommand(command, "SA200GTR;{%s};02;\r", Command.KEY_UNIQUE_ID);
+            case Command.TYPE_OUTPUT_CONTROL:
+                return formatCommand(command, "SA200CMD;{%s};0{%s};Enable{%s}\r",
+                        Command.KEY_UNIQUE_ID, Command.KEY_DATA, Command.KEY_DATA);
```

Comment:
No, both params has to be the same.
Example: 
=> SA200CMD;123456;01;Enable1  ( 01 and Enable1 )
=> SA200CMD;123456;02;Enable2 ( 02 and Enable 2 )

If i make it as two different params would be error prone.